internal/pkg/cli/svc_init.go

```diff
@@ -155,13 +156,15 @@ type initSvcOpts struct {
 	staticAssets []manifest.FileUpload
 
 	// For workspace validation.
+	wsRootGetter      wsRootGetter
```

Comment:
maybe i'm not seeing it - but since the value is cached, it seems like we don't need this on `initSvcOpts`?